packages/ToggleSwitch/ToggleSwitch.jsx

```diff
@@ -150,16 +111,16 @@ ToggleSwitch.propTypes = {
   /** Language provided to the copy prop in TDS ToolTip (en, fr). */
   toolTipCopy: PropTypes.string,
 
-  /** Spinner will be present when selecting toggle */
-  isLoading: PropTypes.bool,
+  /** Communicates a message to assistive technology while spinner is visible. */
+  spinnerLabel: PropTypes.string,
```

Comment:
`spinnerLabel` should be a required prop

Reply:
Done